### PR TITLE
Clean up and redesign loadout optimizer styles

### LINKED ISSUE #8948: LO Attribute collapsing in French at 62px item size and above
### DIM Version

Version 7.50.3.1002173 (beta), built on 07/01/2023 22:24:15

### Browser Details

Edge 108.0.1462.54

### OS Details

Windows 10

### Describe the bug

As the title says, when using French language, even after the fix for the LO Item wrapping, the class item is now placed correctly but the last attribute is still under the others at 62px item size and above. 

Screenshot of French LO at 61px
image

Screenshot of French LO at 62px
image

### Logs

_No response_

### PR DESCRIPTION
**Pull request by bhollis**
Before:
Screenshot 2023-01-08 at 8 31 24 PM

After:
Screenshot 2023-01-08 at 8 31 29 PM

I removed the labels (since they're in the sidebar already) and made the stats line up vertically for easier scanning. Power level is moved to the end as it doesn't matter too much and broke up the tiers list.

Incidentally fixes #8948

### CODE REVIEW

**Comment by nev-r** [comment]
both wrapping and vertical stuff can get pretty weird here when using auto-mods

<hr/>

before
image

after
image

**Comment by ryan-rushton** [comment]
Some random thoughts, not necessarily related to your changes Ben, but just seeing the screen shots nev put up.

- Hmmm, I am not a big fan of those stat mods being in line with the stat tiers. Why didn't we just highlight them in the assigned mods for each item? Maybe a colour overlay or an icon or something on them?
- The information at the top of each row is super busy and lacks structure. This contrasts to the row it items below it. Maybe a richer info display could be available on hover or via button or something. Some thoughts are to put the following in this info display thing
  - the ignored stats and corresponding total,
  - the power level (I personally see this as supplementary information now, it doesn't guide me in any way to choosing a set), 
- Do something with the existing loadout text maybe, it feels out of place and maybe belongs somewhere else, nowhere screams out at me as the right place though

**Comment by bhollis** [comment]
Drat, forgot about the auto mods. Let me see if I can go further here.

**Comment by bhollis** [comment]
Revisiting this now that auto stat mods have been redesigned. The goals here are to reduce clutter, and align stats vertically such that it's easier to scan them as you scroll.

Before:
Screenshot 2023-06-01 at 3 13 35 PM

After:
Screenshot 2023-06-01 at 3 43 14 PM

File: src/app/loadout-builder/generated-sets/SetStats.m.scss [hunk 1]
```diff
@@ -1,25 +1,30 @@
...
-  margin-right: 14px;
   white-space: nowrap;
 
   :global(.app-icon) {
-    font-size: 10px;
-    vertical-align: -0.125em;
-    margin-right: 0;
+    vertical-align: 4px;
+    font-size: 8px;
+    margin-right: 2px;
```
Comment (vivekhnz) [inline src/app/loadout-builder/generated-sets/SetStats.m.scss:27]:
```suggestion
    vertical-align: 3px;
    font-size: 8px;
    margin-right: 1px;
```
This brings the Power 'star' a smidge closer to the text, which is a bit more like how it's rendered in-game.

Before:
image

After:
image

**Review by vivekhnz — COMMENTED** [review]
Nice, much cleaner.

A few observations:
1. If the existing loadout that's matched has a particularly long name, it'll wrap on to the next line. Not an issue on its own, but it also seems to push the `Save Loadout` etc. buttons out to the right.
  image
2. If a set matches an existing loadout and the name wraps, there isn't any vertical space left above the set stats and the set separator line:
  image

Perhaps we could give the 'Existing Loadout' text its own line above the stats to sidestep the wrapping issues?

**Comment by bhollis** [comment]
Good observations. The text wrapping is a problem because of the way the virtual paginator handles calculating heights (I have an idea to replace that, but not quite yet). The heights are basically calculated once and then frozen for all rows. I'll see what I can do to make the buttons not jump over though.

**Review by vivekhnz — APPROVED** [review]
Cool, button placement issues seems to be resolved.

**Pull request merged**